### PR TITLE
removed iterator from utils.py

### PR DESCRIPTION
**Pull request by amit2rockon**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [flake8]
+#ignore line limit error
+ignore = E501,E302,E261,E262,E701,E241,E126,E127,E128,E225
```
Comment (marco-c) [inline setup.cfg]: We don't want to ignore all these errors, but only the line limit error for now.

Comment (amit2rockon) [reply setup.cfg]: okay

**Comment by marco-c** [comment]
Thanks for the PR, but #18 preceded you by a few minutes

**Comment by marco-c** [comment]
If you want, you could start working on #6, which is a good first issue.

**Comment by amit2rockon** [comment]
@marco-c  i have labeled #2  around 3000 data, may i pr for it? , but it is saved in json

**Comment by amit2rockon** [comment]
#6 i am trying for it:)

**Comment by marco-c** [comment]
Yes, you can do it, we can transform it to CSV later on.

**Pull request closed without merging**